frontend/src/App.js

```diff
@@ -22,6 +22,7 @@ import CoursesCreatePage from "main/pages/Courses/PSCourseCreatePage";
 
 import CourseOverTimeIndexPage from "main/pages/CourseOverTime/CourseOverTimeIndexPage";
 
+import SearchByInstructorPage from "main/pages/CourseByInstructor/CourseByInstructor"
```

Comment:
This should be `import CourseByInstructorIndexPage from "main/pages/CourseByInstructor/CourseByInstructorIndexPage"` (so rename `SearchByInstructorPage` to `CourseByInstructorIndexPage` in this file and rename the page itself). Also, moving this one line up would look better.